### PR TITLE
Shrink ray manufacturer text

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Adds a manufacturer-style examine text to the shrink ray

## How This Contributes To The Skyrat Roleplay Experience

it's funny

## Changelog
:cl:
add: The abductor shrink ray now has manufacturer examine text
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
alright now when are we adding ✌︎︎♌︎︎♎︎︎◆︎︎♍︎︎⧫︎︎❄︎♏︎♍︎♒︎ as a guncargo company

File: modular_skyrat/master_files/code/modules/antagonists/abductor/equipment/abduction_gear.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/obj/item/gun/energy/shrink_ray/examine(mob/user)
+	. = ..()
+	. += "<br>It has <b>[span_abductor("✌︎︎♌︎︎♎︎︎◆︎︎♍︎︎⧫︎︎❄︎♏︎♍︎♒︎")]</b> engraved into the photon accelerator."
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/antagonists/abductor/equipment/abduction_gear.dm]: make this a gun company and add it as a var on the gun like it's done elsewhere

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/master_files/code/modules/antagonists/abductor/equipment/abduction_gear.dm (hunk 1 above)
Comment (Zonespace27) [reply modular_skyrat/master_files/code/modules/antagonists/abductor/equipment/abduction_gear.dm]: and add it to the laser gun ayys have as well while you're at it

**Comment by Gandalf2k15** [comment]
Photon accelerator = torch

**Pull request merged**